docs/references/_attachments/ic.did

```diff
@@ -54,9 +62,10 @@ type change_details = variant {
       snapshot_id : snapshot_id;
       taken_at_timestamp : nat64;
     };
-    controllers_change : record {
-        controllers : vec principal;
-    };
```

Comment:
Do we want to migrate existing instances of this variant in the state to the new variant (e.g., upon replica upgrade)? If not, then we should keep this variant in the spec.